backend/attendance-service/src/main/java/org/egov/repository/AttendanceAttendeeRepository.java

```diff
@@ -0,0 +1,31 @@
+package org.egov.repository;
+
+import org.egov.repository.querybuilder.AttendanceAttendeeQueryBuilder;
+import org.egov.repository.rowmapper.AttendanceAttendeeRowMapper;
+import org.egov.web.models.AttendanceAttendeeSearchCriteria;
+import org.egov.web.models.IndividualEntry;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Repository
+public class AttendanceAttendeeRepository {
```

Comment:
Rename it to just AttendeeRepository

Reply:
From all the repository classes word Attendance has been removed

Reply:
Same applies for other classes also: RowMapper, QueryBuilder, Service, etc. 
We have 4 entities:
1. AttendanceRegister
2. Attendee
3. Staff
4. AttendanceLog
Use the above 4 prefixes for every class name.